app/controllers/provider/admin/backend_apis_controller.rb

```diff
@@ -42,7 +42,12 @@ def update
   end
 
   def destroy
-    # TODO
+    if @backend_api.services.empty? && @backend_api.destroy
+      redirect_to provider_admin_dashboard_path, notice: 'Backend API deleted'
+    else
+      flash[:error] = "Backend API could not be deleted"
```

Comment:
Imo the flash message should be in i18n 😄 but it isn't a huge problem if it is here 👍

Reply:
If I changed it, I would have to change all the messages in the controller. I suggest we do that in a different PR.